src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -51,6 +59,13 @@ const messages: { [messageName: string]: MessageDescriptor } = defineMessages({
   'sonarr-scan': 'Sonarr Scan',
   'download-sync': 'Download Sync',
   'download-sync-reset': 'Download Sync Reset',
+  editJobSchedule: 'Modify Job Schedule',
+  jobScheduleEditSaved: 'Job schedule modified',
+  jobScheduleEditFailed:
+    'Something went wrong while modifying the job schedule',
+  editJobSchedulePrompt: 'Select job schedule',
+  editJobScheduleSelector:
+    'Every {jobScheduleMinutes} minute(s) or {jobScheduleHours} hour(s)',
```

Comment:
This is confusing and not very user-friendly.


How about something like:
```
Every [ <number>  v] [ minutes  v]
                     [ hour(s)   ]
```

(We should not allow users to run anything more often than once every 5 minutes, by the way.)

Reply:
In fact, we should probably enforce some "sane" limits here so that people don't try to run full Plex scans very few minutes.

Perhaps we should have some jobs for which users can choose a "every X hours" job frequency, and others for which they can select a "every X minutes" job frequency.

Reply:
Do we want to restrict the frequency for jobs that can only be ran "every X hours" to something else than 1 to 100? I was thinking 72 would be a reasonable maximum.

Reply:
Yeah that sounds reasonable for "long" jobs.

About `Download Sync`, I'm not really sure what would be right way to deal with this. Making it uneditable is a good enough solution imo since that job barely requires any computing power unlike a `Plex Recently Added Scan` for example.

Reply:
Yeah that's also my thought.  There really shouldn't be any need to modify that one.  So I guess we'll have three job types, "fixed" or "static" (unsure the best name for this), "short," and "long"? 🤔